src/crimson/osd/objclass.cc

```diff
@@ -552,17 +552,6 @@ uint64_t cls_get_osd_min_alloc_size(cls_method_context_t hctx) {
   return 4096;
 }
 
-int cls_cxx_gather(cls_method_context_t hctx, const std::set<std::string> &src_objs, const std::string& pool,
-		   const char *cls, const char *method, bufferlist& inbl)
-{
-  return 0;
-}
-
-int cls_cxx_get_gathered_data(cls_method_context_t hctx, std::map<std::string, bufferlist> *results)
```

Comment:
I'm fine with removing these functions in crimson assuming the plugins are still loadable (I bet they will be).